Add example to use "tenant_route()" helper. (#80)
stancl/tenancy-docs: source/docs/v3/features/cross-domain-redirect.blade.md

```php
return redirect()->route('home')->domain($domain);
```

> You can also use the `tenant_route()` helper to redirect users between domains. Which also works when redirecting the user from the central domain to the tenant domain.

Reviewer: ```suggestion
You can also use the `tenant_route()` helper to redirect users to another domain.
```
